specification/eventgrid/data-plane/Microsoft.ContainerRegistry/stable/2018-01-01/ContainerRegistry.json

```diff
@@ -0,0 +1,145 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "2018-01-01",
+        "title": "Schema of Azure Container Registry events published to Azure Event Grid",
+        "description": "Describes the schema of the Azure Container Registry events published to Azure Event Grid. This corresponds to the Data property of an EventGridEvent."
+    },
+    "paths": {},
+    "definitions": {
+        "ContainerRegistryImagePushedEventData": {
+            "description": "Schema of the Data property of an EventGridEvent for a Microsoft.ContainerRegistry.ImagePushed event.",
+            "allOf": [
+                {
+                    "$ref": "#/definitions/EventContent"
```

Comment:
For all the types referenced here (EventContent, Actor, Target, etc.), it looks to me that they should be qualified with the prefix "ContainerRegistry" as all these types will get generated under the same namespace Microsoft.Azure.EventGrid.Models. @lmazuel, what do you think, is there a better approach?

Reply:
`EventContent` can be prefixed with `ContainerRegistry` for sure. We had a similar issue with the naming here in our own models that types such as `Target`, `Request`, and `Source` are from standard `docker` notification schema but they may conflict with other models in the same namespace.

Are you generating data-plane models into SDKs?